rio_tiler/io/cogeo.py

```diff
@@ -94,6 +94,10 @@ class COGReader(BaseReader):
     minzoom: int = attr.ib(default=None)
     maxzoom: int = attr.ib(default=None)
     colormap: Dict = attr.ib(default=None)
+    tile_coord_width: Optional[Union[int, float]] = attr.ib(
+        default=constants.WEB_MERCATOR_COORD_WIDTH
+    )
+    tile_opp_crs: Optional[CRS] = attr.ib(default=constants.WEB_MERCATOR_OPP_CRS)
```

Comment:
Not sure to understand this. I think this will make working with other TMS quite complicated 🤷‍♂️

Reply:
I too think to be quite complicated.
But, opposite site CRS was necessary to get `get_vrt_transform` to calculate Affine correctly.
https://github.com/cogeotiff/rio-tiler/blob/982811a8054f402d75d5d4d924b0d05d72846155/rio_tiler/reader.py#L246

Do you have a better idea?

Reply:
I don't have a better idea 🙁 because I yet don't fully understand the underlying need to change this (I'll comment in the issue)